xamarin/Xamarin.Forms
# [Android] hide keyboard after tapping on pickers

### Description of Change ###



### Issues Resolved ### 


- fixes #6235 

### API Changes ###

 
 None

### Platforms Affected ### 

- Android

### Behavioral/Visual Changes ###


None

### Before/After Screenshots ### 


Not applicable

### Testing Procedure ###

- open Visual Gallery
- select editor, open keyboard
- select any picker
- the keyboard should be closed

### PR Checklist ###
